packages/components/src/components/tooltip/tooltip.stories.ts

```diff
@@ -65,198 +70,81 @@ export const Default = {
   }
 };
 
-export const Placement = {
-  parameters: { controls: { exclude: 'placement' } },
-  render: (args: any) => {
-    return html`<div class="placement-story">
-      ${['top', 'bottom'].map(value =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'attribute',
-              name: 'placement',
-              values: [value, `${value}-start`, `${value}-end`]
-            }
-          },
-          args,
-          constants: [
-            {
-              type: 'template',
-              name: 'placement',
-              value: `<div class="template-height template-position">%TEMPLATE%</div>`
-            }
-          ]
-        })
-      )}
-    </div> `;
-  }
+/**
+ * Use the `size` attribute to change the tooltip trigger size.
+ *
+ * - `large` is the default tooltip trigger size.
+ * - `small` can be used as an alternative.
+ */
+export const Size = {
+  render: () => html`
+    <div class="flex items-center gap-12">
+      <sd-tooltip content="Lorem ipsum" placement="bottom" size="lg" trigger="click focus"></sd-tooltip>
+      <sd-tooltip content="Lorem ipsum" placement="bottom" size="sm" trigger="click focus"></sd-tooltip>
+    </div>
+  `
 };
 
-export const Size = {
-  parameters: { controls: { exclude: 'size' } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'size'
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div class="template-position">%TEMPLATE%</div>' },
-        {
-          type: 'attribute',
-          name: 'placement',
-          value: 'bottom'
-        }
-      ]
-    })
+/**
+ * Use the `placement` attribute to set the placement.
+ *
+ * - `top-start`: displays tooltip above the trigger element and aligns arrow to the start of the tooltip container
```

Comment:
Can you replace Lorem Impsum with the same copy as Figma please?

Reply:
Also when scrolling on the sample the bottom line of components overlaps the upper one. Can you have a look?